fitnessllm_dataplatform/task_handler.py

```
@@ -0,0 +1,42 @@
+from cloudpathlib import GSClient
+
+from fitnessllm_dataplatform.entities.enums import DynamicEnum
+from fitnessllm_dataplatform.entities.enums import FitnessLLMDataSource
+from fitnessllm_dataplatform.stream.strava.etl_utils import load_json_into_bq
+from fitnessllm_dataplatform.stream.strava.services.api_interface import StravaAPIInterface
+from fitnessllm_dataplatform.utils.cloud_utils import get_secret
+from fitnessllm_dataplatform.utils.logging_utils import logger
+from fitnessllm_dataplatform.utils.task_utils import load_into_env_vars
+
+
+class Startup(object):
```

Ruff (0.8.2)

Class `Startup` inherits from `object`

Remove `object` inheritance

(UP004)